zebra-state/src/lib.rs

```diff
@@ -16,12 +16,20 @@
 #![allow(clippy::try_err)]
 use serde::{Deserialize, Serialize};
 use std::path::PathBuf;
-use std::sync::Arc;
-use zebra_chain::block::{Block, BlockHeaderHash};
+use std::{iter, sync::Arc};
+use zebra_chain::{
+    block::{Block, BlockHeaderHash},
+    types::BlockHeight,
+};
 
 pub mod in_memory;
 pub mod on_disk;
 
+const GENESIS: BlockHeaderHash = BlockHeaderHash([
+    8, 206, 61, 151, 49, 176, 0, 192, 131, 56, 69, 92, 138, 74, 107, 208, 93, 161, 110, 38, 177,
+    29, 170, 27, 145, 113, 132, 236, 232, 15, 4, 0,
+]);
+
```

Comment:
Issue: please use the new `zebra_consensus::parameters::genesis_hash()` function to get the genesis hash for the current network.

Since zebra-consensus already depends on zebra-state, it might be better to have zebrad send the genesis hash as part of the request. (And zebrad will have to know the genesis hash anyway.)